okta-hosted-login/src/Navbar.jsx

```diff
@@ -13,13 +13,72 @@
 import { useOktaAuth } from '@okta/okta-react';
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Container, Icon, Image, Menu } from 'semantic-ui-react';
+import { Container, Icon, Image, Menu, Modal } from 'semantic-ui-react';
+import config from './config';
 
 const Navbar = () => {
+  const [errorModalOpen, setErrorModalOpen] = React.useState(false);
   const { authState, oktaAuth } = useOktaAuth();
 
+  // Note: Can't distinguish CORS error from other network errors
```

Comment:
This is actually a fairly common problem - A dev will have something misconfigured (such as the wrong issuer, or wrong application type) and a call comes back with an HTTP status error.  That wouldn't normally trip this, but many errors come back WITHOUT the CORS headers.  

So they see a CORS error, but the real problem not CORS at all, and their CORS settings can be completely correct.

I'm not sure how to summarize this sufficiently for the below message, and I think your message is an improvement over not getting any info, so I recommend running with what you have but considering if we want to modify the message as we see how it works in the field.